frontend/src/pages/org/archived-item-detail/archived-item-detail.ts

```diff
@@ -275,7 +275,21 @@ export class ArchivedItemDetail extends TailwindElement {
         ]);
         break;
       case "files":
-        sectionContent = this.renderPanel(msg("Files"), this.renderFiles());
+        sectionContent = this.renderPanel(
+          html` ${this.renderTitle(msg("Files"))}
+            <sl-tooltip content=${msg("Download all files as a single WACZ")}>
+              <sl-button
+                href=${`/api/orgs/${this.orgId}/all-crawls/${this.crawlId}/download?auth_bearer=${authToken}`}
+                download}
+                size="small"
+                variant="primary"
+              >
+                <sl-icon slot="prefix" name="cloud-download"></sl-icon>
+                ${msg("Download")}
```

Comment:
Would _probably_ like to keep this labeled the same for consistency?  Also considered "Download all" but that makes it sound like they'll all be started individually.  Tooltip is good!

```suggestion
                ${msg("Download Item")}
```